deployments/systemd/config-default.yaml

```diff
@@ -151,7 +151,55 @@ mig-configs:
       mig-devices:
         "7g.96gb": 1
 
-  # H100-80GB, H800-80GB, A100-40GB, A100-80GB, A800-40GB, A800-80GB, A30-24GB, PG506-96GB
+  # H100-94GB
+  all-1g.11gb:
+    - device-filter: "0x232110DE"
+      devices: all
+      mig-enabled: true
+      mig-devices:
+        "1g.11gb": 7
+
+    - device-filter: "0x232110DE"
+      devices: all
+      mig-enabled: true
+      mig-devices:
+        "1g.11gb+me": 1
+
+    - device-filter: "0x232110DE"
+      devices: all
+      mig-enabled: true
+      mig-devices:
+        "1g.22gb": 4
```

Comment:
2 out of 3 of these seem to be missing the top-level name

Reply:
oh thank you for catching that